contracts/src/v0.8/vrf/test/VRFV2PlusWrapper_Optimism.t.sol

```diff
@@ -0,0 +1,216 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+import {BaseTest} from "./BaseTest.t.sol";
+import {MockLinkToken} from "../../mocks/MockLinkToken.sol";
+import {MockV3Aggregator} from "../../tests/MockV3Aggregator.sol";
+import {ExposedVRFCoordinatorV2_5_Optimism} from "../dev/testhelpers/ExposedVRFCoordinatorV2_5_Optimism.sol";
+import {VRFV2PlusWrapper_Optimism} from "../dev/VRFV2PlusWrapper_Optimism.sol";
+import {OptimismL1Fees} from "../dev/OptimismL1Fees.sol";
+import {GasPriceOracle as OVM_GasPriceOracle} from "../../vendor/@eth-optimism/contracts-bedrock/v0.17.3/src/L2/GasPriceOracle.sol";
+import {VmSafe} from "forge-std/Vm.sol";
+
+contract VRFV2PlusWrapperTest is BaseTest {
```

Comment:
Its probably better to mention that this test covers Base network as well

Reply:
Done